### ISSUE TITLE
Сделать бота более приветливым

### ISSUE THREAD

**Issue by petr-kalinin** [petr-kalinin/algoprog#286]
Надо как минимум сделать так, чтобы бот как-то реагировал на начало диалога со стороны пользователя, т.е. реагировать на команду /start. Бот должен проверять, нашел ли он этого пользователя среди пользователей алгопрога, и отвечать типа
> Здравствуйте! Я определил вас как пользователя Василий Пупкин, https://algoprog.ru/user/123456
или
> Здравствуйте! Я не понимаю, кто вы. Укажите в вашем профиле на алгопроге id телеграмма 123456789, и повторно отправьте мне команду /start

Возможно, стоит отвечать так на любые сообщения от пользователя, а не только на /start.

Также можно заодно добавить какие-нибудь еще команды.

**Comment by UlinEgor** [comment]
> Возможно, стоит отвечать так на любые сообщения от пользователя, а не только на /start.

Возможно для пользователей будет удобнее кнопка, которая будет выполнять "проверку" того, что бот подключен, и назвать её по типу "Проверить работоспособность" (или как-то "поприятнее" для пользователей). И при её нажатии просто запускать проверку (можно писать сообщение, аналогичное стартовому).

**Comment by petr-kalinin** [comment]
Ты имеешь в виду кнопку на сайте, или кнопку в боте?

Кнопка в боте -- это по сути то же, что я и предлагаю в этом тикете (только постоянную кнопку в телеграмме не сделаешь, поэтому надо делать команду).

Кнопка на сайте имеет ту проблему, что если пользователь что-то сделал не так (не начал диалог с ботом или не прописал id), то мы не сможем объяснить пользователю, что именно не так. Команда в боте, во-первых, гарантирует, что пользователь начал диалог, а во-вторых может сразу пользователю сообщить его id. Кнопка на сайте ничего этого сделать не может.

**Comment by petr-kalinin** [comment]
Конечно, все, что умеет телеграм, можно сделать в любом языке, потому что в конечном счете api телеграмма -- это просто сетевые запросы с нужным содержимым. Поэтому что питон, что coffeescript это умеют.

Но я бы все-таки делал бы команду, потому что кнопка 99% времени не будет нужна: как только человек все правильно прописал на сайте, кнопка ему не будет нужна. Ну и плюс команд можно будет сделать больше, если придумаем, что еще потребуется. Например, можно добавить команду на вывод последних комментариев, или даже на сброс пароля. И команды в боте тоже неплохо видны, в строке ввода сообщения отображается кнопка "меню" со списком команд (см, например, как это сделано у BotFather)
